helpers/cp.go

```diff
@@ -57,6 +59,11 @@ func doOpenFile(name string, flag int, perm os.FileMode) (fileish, error) {
 
 // CopyFile copies src to dst
 func CopyFile(src, dst string, flags CopyFlag) (err error) {
+	// we do not preserve all yet here natively so just use "cp"
```

Comment:
This comment is somewhat confusing to me. I assume you meant to say that go doesn't have a "cp -p" like function so we just use cp directly but I had to read that a few times to be sure.